### ISSUE TITLE
Some icons don't work properly.

### ISSUE THREAD

**Issue by giwhub** [moka-project/moka-icon-theme#226]
Os:Debian testing
De:Cinnamon 3.0

Gedit:
![screenshot from 2016-05-28 08-37-09](https://cloud.githubusercontent.com/assets/14066437/15624059/de2cd760-24b0-11e6-96ad-3e958fbacc7e.png)

Open:
![screenshot from 2016-05-28 08-42-21](https://cloud.githubusercontent.com/assets/14066437/15624068/f7e3f7c4-24b0-11e6-96fa-c4a8b731793f.png)

Search:
![screenshot from 2016-05-28 08-39-24](https://cloud.githubusercontent.com/assets/14066437/15624070/022989ba-24b1-11e6-9c31-f0cf67529de5.png)

Open as root:
![screenshot from 2016-05-28 08-43-05](https://cloud.githubusercontent.com/assets/14066437/15624083/2f2baa88-24b1-11e6-8c5a-6ed2bc2845c2.png)

**Comment by giwhub** [comment]
I solved them in my ways below,but still hope offical fixs.

Gedit:
I copied "tab-new-symbolic.svg" from another icon theme and put it into Faba/16x16/actions,it works.

Open:
I made a link of "system-file-manager.png" which located in Moka/16x16/apps,and renamed the name of the link as "gtk-open" then put it into Moka/16x16/actions,it works.

Search:
I copied "edit-clear-symbolic.svg" from another icon theme and put it into Faba/16x16/actions,it works.

Open as root:
I copied "dialog-password.svg" from Faba/48x48/status and put it into Faba/24x24/status,it works.
